webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -41,6 +42,19 @@
       <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signature }} ]</h2>
     </div>
 
+    <div class="protected-info">
+      {% if your_crash %}
+        You are seeing public and protected data for your crash.
```

Comment:
This bit is only in the report view because users sometimes look at their own reports.